doc/classes/SpatialMaterial.xml

```diff
@@ -87,6 +87,10 @@
 		<member name="ao_texture_channel" type="int" setter="set_ao_texture_channel" getter="get_ao_texture_channel" enum="SpatialMaterial.TextureChannel">
 			Specifies the channel of the [member ao_texture] in which the ambient occlusion information is stored. This is useful when you store the information for multiple effects in a single texture. For example if you stored metallic in the red channel, roughness in the blue, and ambient occlusion in the green you could reduce the number of textures you use.
 		</member>
+		<member name="async_mode" type="int" setter="set_async_mode" getter="get_async_mode" enum="SpatialMaterial.AsyncMode" default="0">
+			If [member ProjectSettings.rendering/gles3/shaders/async_compile_enabled] is [code]true[/code], this determines how this material must behave in regards to asynchronous shader compilation.
```

Comment:
```suggestion
			If [member ProjectSettings.rendering/gles3/shaders/shader_compilation_mode] is [code]true[/code], this determines how this material must behave in regards to asynchronous shader compilation.
```

Reply:
Coming in next push.

Reply:
Also changed the _is [code]true[/code]_ part, since it's an enum now.